### PR TITLE
[docs] Apache superset documentation to third party tools section

### PR DESCRIPTION
**Pull request by aatrey-yb**

### CODE REVIEW

**Comment by aatrey-yb** [comment]
@netlify /latest/tools/superset/

**Comment by aatrey-yb** [comment]
@netlify /latest/tools/superset/

File: docs/content/latest/tools/superset.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Use Apache Superset with YugabyteDB YSQL
+headerTitle: Apache Superset
+linkTitle: Apache Superset
+description: Use Apache Superset to explore and visulize data in YugabyteDB.
+menu:
+  latest:
+    identifier: superset
+    parent: tools
+    weight: 2794
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:10]: Let's put this right before Arctype in the left nav
```suggestion
    weight: 2709
```

File: docs/content/latest/tools/superset.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+showAsideToc: true
+---
+
+[Apache Superset](https://superset.apache.org/) is a fast, lightweight and intuitive open-source data exploration and visualization tool that helps you query your data stored in YugabyteDB and visualize it using simple line charts to highly detailed geospatial charts.
+
+![Superset Dashboard](/images/develop/tools/superset/dashboard.png)
+
+## Before you begin
+
+Your YugabyteDB cluster should be up and running. If you're new to YugabyteDB, create a local cluster in less than five minutes following the steps in [Quick Start](../../quick-start/install). You can also get started with the free tier of [YugabyteDB Fully-Managed Cloud](https://www.yugabyte.com/cloud/). You also need to load a relevant database in your YugabyteDB for exploration and visualizaation. If using yugabyted, you can load the Northwind sample database with the ./bin/yugabyted demo connect command from your shell or you can follow [instructions here](https://docs.yugabyte.com/latest/sample-data/northwind/).
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:21]:
```suggestion
Your YugabyteDB cluster should be up and running. If you're new to YugabyteDB, create a local cluster in less than five minutes following the steps in [Quick Start](../../quick-start/install). You can also get started with the free tier of [YugabyteDB Fully-Managed Cloud](https://www.yugabyte.com/cloud/).

You also need to load a relevant database in your YugabyteDB for exploration and visualization. You can load the Northwind sample database with the `./bin/yugabyted demo connect` command from your shell, or you can follow the [instructions here](https://docs.yugabyte.com/latest/sample-data/northwind/).
```

File: docs/content/latest/tools/superset.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+You also need to install Apache Superset to explore and visualize your data. You can install Superset using [Docker Compose](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose) (recommended) or from scratch using [python install (PIP)](https://superset.apache.org/docs/installation/installing-superset-from-scratch).
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:23]: What version of Python is required for this? If it's 3, as I suspect, change `(pip)` to `(pip3)` as a little clue for the reader.

```suggestion
You also need to install Apache Superset to explore and visualize your data. You can install Superset using [Docker Compose](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose) (recommended) or from scratch using [Python (pip)](https://superset.apache.org/docs/installation/installing-superset-from-scratch).
```

File: docs/content/latest/tools/superset.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Connecting Apache Superset with YugabyteDB
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:25]:
```suggestion
## Connect Apache Superset to YugabyteDB
```

File: docs/content/latest/tools/superset.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+After successful installation, launch Superset in the browser at http://<hostname/IP>:8088. In case of local installation on your laptop, navigate to localhost:8088 or 127.0.0.1:8088. Superset comes with standard PostgreSQL driver that also connects to YugabyteDB. You can also manually install [psycopg2 driver](https://www.psycopg.org/docs/) to connect to YugabyteDB.
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:27]:
```suggestion
After successful installation, launch Superset in your browser at `http://<hostname-or-IP-address>:8088`. If you've installed on your local computer, navigate to `localhost:8088` or `127.0.0.1:8088`. Superset comes with a standard PostgreSQL driver that also connects to YugabyteDB. You can also manually install the [psycopg2 driver](https://www.psycopg.org/docs/) to connect to YugabyteDB.
```

File: docs/content/latest/tools/superset.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+Follow these steps to connect Apache Superset to YugabyteDB:
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:29]:
```suggestion
To connect Apache Superset to YugabyteDB:
```

File: docs/content/latest/tools/superset.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Navigate to Data → Databases → + Databases and choose "PostgreSQL" from the “Connect a database” menu as shown below.
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:31]: Don't use the arrows if possible. They won't render correctly on all browsers/machines. The usual convention is > instead.
```suggestion
1. Navigate to Data > Databases > + Databases, and choose "PostgreSQL" from the “Connect a database” menu.
```

File: docs/content/latest/tools/superset.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+2. Enter the hostname or IP Address of your YB-TServer with standard credentials and click "Finish".
+
+![Database Connection Credentials](/images/develop/tools/superset/connect-ybdb.png)
+
+{{< note title="Note" >}}
+As of Docker version 18.03, the host.docker.internal hostname connects to your Docker host from inside a Docker container.
+{{< /note >}}
+
+3. You should now be connected to the YugabyteDB and to verify you can check the availabe databases and schemas under "Data". Navigate to Data → Datasets and click "+Datasets" and in the dialog configure as shown below. Dropdown selectors would show the datasbes and schemas available for explore and visualize operations.
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:43]: Number everything with `1.` — it makes editing lists a lot easier later on!

```suggestion
1. Enter your YugabyteDB tablet server's hostname or IP address with standard credentials and click "Finish".

    ![Database Connection Credentials](/images/develop/tools/superset/connect-ybdb.png)

    {{< note title="Note" >}}
As of Docker version 18.03, the `host.docker.internal` hostname connects to your Docker host from inside a Docker container.
    {{< /note >}}

1. Verify that you can access the available databases and schemas under "Data". Navigate to Data > Datasets, and click "+Datasets".

    The dropdown list should show the databases and schemas available to explore and visualize.
```

File: docs/content/latest/tools/superset.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+    ![Loading Datasets](/images/develop/tools/superset/load-dataset.png)
+
+You've successfully created a connection to your YugabyteDB database, and you can now start exploring and visualizing your databases using Apache Superset.
+
+## What's Next
+
+Superset is a rich, open-source data exploration and visualization platform that allows Business Analysts, Data Scientists and Developers to quickly explore and visualize data stored in their databases and data warehouses. You can explore data without writing complex SQL queries, create rich reports and custom dashboards to visualize this data and get insights quickly.
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:51]: Don't capitalize things unless they're proper nouns or trademarked terms...

```suggestion
Superset is a rich, open-source data exploration and visualization platform that allows business analysts, data scientists, and developers to quickly explore and visualize data stored in their databases and data warehouses. You can explore data without writing complex SQL queries, create rich reports and custom dashboards to visualize this data, and get insights quickly.
```

File: docs/content/latest/tools/superset.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+Refer to Apache Superset [documentation](https://superset.apache.org/docs/creating-charts-dashboards/exploring-data#exploring-data-in-superset) to learn more about Superset's data exploration capabilities. Also, in case you are creating your first dashboard using Superset then check out [instructions for data analysis and exploration workflow](https://superset.apache.org/docs/creating-charts-dashboards/creating-your-first-dashboard/).
```
Comment (polarweasel) [inline docs/content/latest/tools/superset.md:53]:
```suggestion
Refer to the Apache [Superset documentation](https://superset.apache.org/docs/creating-charts-dashboards/exploring-data#exploring-data-in-superset) to learn more about Superset's data exploration capabilities. If you're creating your first dashboard using Superset, check out the [data analysis and exploration workflow](https://superset.apache.org/docs/creating-charts-dashboards/creating-your-first-dashboard/).
```

**Review by polarweasel — CHANGES_REQUESTED** [review]

File: docs/content/latest/tools/superset.md (hunk 8 above)
Comment (polarweasel) [reply docs/content/latest/tools/superset.md:43]: (Also, the weird formatting of the note is correct. If you indent the _content_, it'll render as a code block!)

**Review by polarweasel — APPROVED** [review]

**Pull request merged**